Windows/MainWindow.cpp

```diff
@@ -281,6 +283,10 @@ namespace MainWindow
 			NativeMessageReceived("gpu resized", "");
 		}
 
+		if (screenManager) {
+			screenManager->RecreateAllViews();
```

Comment:
Well that's a fix for the issue which messes up both this and touch control layout screens since they aren't recreated whenever you resize window as on Henrik's screenshot above. Not really sure how to deal with it differently.